### PR TITLE
Added portal-authenticated-mode variable for portal authentication mode

### PR DESCRIPTION
**Pull request by dzdimov**
Added a portal-authenticated-mode variable for the portal authentication mode and set a default value of false if the fact is not set.
After this, we will need to update the templates to include this fact if we want to set the portal to authenticated mode after installation

### CODE REVIEW

File: vagrant/provisioning/roles/foia/templates/arkcase-portal-server.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ portal.uiApplicationUrl: "https://{{ external_host }}/foia"
 portal.uiApplicationHost: "https://{{ external_host }}"
 portal.id: {{ 9999999999999999999999 | random | to_uuid }}
 
-portal.authenticatedMode: false
+portal.authenticatedMode: {{ portal-authenticated-mode | default (false) }}
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/foia/templates/arkcase-portal-server.yaml]: fact names cannot include dashes, you have to use underscores instead

File: vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml [hunk 2]
```diff
@@ -25,196 +25,9 @@
...
-  become: yes
-  become_user: pentaho
-  command:
-    cmd: 'curl -k -v -H "Content-Type: multipart/form-data" -X PUT -F uploadInput=@{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}/mondrian_schema/{{ foia_mondrian_schema_file }} -F overwrite=true -F xmlaEnabledFlag=false -F parameters="Datasource=MariaDB Local" -u {{ ldap_user_prefix }}arkcase-admin:{{ arkcase_admin_password }} https://{{ internal_host }}:2002/pentaho/plugin/data-access/api/datasource/analysis/catalog/{{ foia_mondrian_schema_file | replace(".xml", "") }}'
-    warn: no
-
-#########################################################
-# Deploy the reports 
-#########################################################
+- set_fact:
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml:28]: I'm going to guess you didn't mean to include these foia-analytical-report changes?

**Comment by dzdimov** [comment]
Somehow some other changes I was working on got into this MR. 
Closing this one and will open a new one

File: vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml (hunk 2 above)
Comment (dzdimov) [reply vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml:28]: That's what you get when you type git commit -am 🥇

**Pull request closed without merging**